packages/apps/tools/src/utils/network.ts

```diff
@@ -0,0 +1,53 @@
+import type { DefinedNetwork, Network } from '@/constants/kadena';
+import { kadenaConstants } from '@/constants/kadena';
+
+interface IApiHostData {
+  networkId: string;
+  chainId: string;
+}
+
+export interface INetworkDto {
+  networkId: Network;
+  label: string;
+  API: string;
+  apiHost: ({ networkId, chainId }: IApiHostData) => string;
+}
+
+const isNetwork = (x: any): x is Network =>
+  ['mainnet01', 'testnet04'].includes(x);
+
+export const getConfigNetworkNames = (): DefinedNetwork[] => {
+  return Object.keys(kadenaConstants).filter((key) =>
+    isNetwork(key),
+  ) as DefinedNetwork[];
+};
```

Comment:
Does it only include predefined networks or should it include custom networks here?

Reply:
Here I'm only fetching the predefined config networks and in the /utils/network.ts all networks are handled